geoh5py/shared/validators.py

```diff
@@ -105,10 +105,31 @@ def to_class(
     return out
 
 
-def empty_string_to_uid(value):
+def empty_string_to_none(value):
     """Promote empty string to uid, and pass all other values."""
     if value == "":
-        return UUID("00000000-0000-0000-0000-000000000000")
+        return None
+    return value
+
+
+def none_to_empty_string(value):
+    """None transforms to empty string for serialization."""
+    if value is None:
+        return ""
+    return value
+
+
+def types_to_string(types: list) -> list[str] | str:
+    if len(types) > 1:
+        return [f"{{{k.default_type_uid()}}}" for k in types]
+    return f"{{{types[0].default_type_uid()}}}"
```

Comment:
```suggestion
        return [str(k.default_type_uid()) for k in types]
    return str(types[0].default_type_uid())
```
Why so many braces?

Reply:
It's to make the uuid string like "{blahblahblah}"

Reply:
I don't mind breaking the mold, but it's how we've formatted the uid's up to now.